### PR TITLE
fix docs for migrate diff

### PR DESCRIPTION
**Pull request by Jolg42**
I found 2 problems while looking at this issue, https://github.com/prisma/prisma/issues/14950#issuecomment-1691203291, and then looking at the docs

- the path to the migrations' directory should be `./prisma/migrations` or it would error with a misleading message at the moment.
- the missing required `--shadow-database-url` in that case

### CODE REVIEW

File: content/300-guides/025-migrate/800-production-troubleshooting.mdx [hunk 1]
```diff
@@ -135,11 +135,16 @@ In this case, you need to create a migration that takes your production database
...
-     npx prisma migrate diff --from-url "$DATABASE_URL_PROD" --to-migrations ./migrations --script > backward.sql
+    ```terminal wrap
+     npx prisma migrate diff \
+      --from-url "$DATABASE_URL_PROD" \
+      --to-migrations ./prisma/migrations \ 
+      --shadow-database-url $SHADOW_DATABASE_URL \
+      --script > backward.sql
     ```
-
+
```
Comment (ruheni) [inline content/300-guides/025-migrate/800-production-troubleshooting.mdx]:
```suggestion

```

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx [hunk 2]
```diff
@@ -13,6 +13,8 @@ Some development-focused commands for relational databases of Prisma Migrate use
 
 The shadow database is **created and deleted automatically**\* each time you run a development-focused command and is primarily used to **detect problems** such as schema drift.
 
+[`migrate diff` command](/reference/api-reference/command-reference#migrate-diff) also requires a shadow database when diffing against a local `migrations` directory.
```
Comment (janpio) [inline content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]: Can we list the arguments that is the case for?

File: content/300-guides/025-migrate/800-production-troubleshooting.mdx [hunk 3]
```diff
@@ -135,11 +135,15 @@ In this case, you need to create a migration that takes your production database
...
   - Run the following `prisma migrate diff` command:
 
-    ```bash
-     npx prisma migrate diff --from-url "$DATABASE_URL_PROD" --to-migrations ./migrations --script > backward.sql
+    ```terminal wrap
+     npx prisma migrate diff \
+      --from-url "$DATABASE_URL_PROD" \
+      --to-migrations ./prisma/migrations \ 
+      --shadow-database-url $SHADOW_DATABASE_URL \
+      --script > backward.sql
```
Comment (janpio) [inline content/300-guides/025-migrate/800-production-troubleshooting.mdx:143]: I don't this notation works on Windows. (But I see we already use that in other places as well 🤷)

Comment (Jolg42) [reply content/300-guides/025-migrate/800-production-troubleshooting.mdx:143]: If using `cmd.exe` it won't work indeed but given that it's in many places like this it should be its own issue.
See https://superuser.com/a/1222011 the continuation character is different.

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx (hunk 2 above)
Comment (Jolg42) [reply content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]: Only `--to/from-migrations` like listed in the reference.

File: content/300-guides/025-migrate/800-production-troubleshooting.mdx (hunk 3 above)
Comment (janpio) [reply content/300-guides/025-migrate/800-production-troubleshooting.mdx:143]: TIL

**Comment by Jolg42** [comment]
@jharrell / @ruheni / @janpio  Please review so we can merge 😉

**Review by jharrell — APPROVED** [review]

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx (hunk 2 above)
Comment (janpio) [reply content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]:
```suggestion
[`migrate diff` command](/reference/api-reference/command-reference#migrate-diff) also requires a shadow database when diffing against a local `migrations` directory with `--from-migrations` or `--to-migrations`.
```

**Review by janpio — APPROVED** [review]

**Pull request merged**